.gitlab-ci/base_jobs.libsonnet

```diff
@@ -22,16 +22,16 @@ local appr = utils.appr;
 
     AlmTest: {
```

Comment:
OlmTest?

Reply:
Deleted - I removed the unit tests from gitlab since they run in the other ci system now